lib/order.rb

```diff
@@ -0,0 +1,36 @@
+require './lib/takeaway'
+require './lib/message'
+require 'twilio-ruby'
+
+class Order
+
+  attr_reader :order
+
+  def initialize
+    @order = {}
+  end
+
+  def add(item)
+    if @order[item] == nil
+      @order[item] = 1
+    else
+      count = @order[item]
+      @order[item] = count + 1
+    end
```

Comment:
Check out the conditional assignment operator, it might save space here: https://stackoverflow.com/a/14697343